core/docker/default/etc/jvm.config

```diff
@@ -1,7 +1,7 @@
 -server
--Xmx1G
+-XX:InitialRAMPercentage=80
+-XX:MaxRAMPercentage=80
 -XX:-UseBiasedLocking
```

Comment:
We should remove this everywhere since it is deprecated and disabled by default in https://openjdk.org/jeps/374